### PR TITLE
add landing page for hopsworks ai and reorganize sections

### PR DESCRIPTION
**Pull request by berthoug**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -9,9 +9,50 @@ repo_url: https://github.com/logicalclocks/feature-store-api
...
+      - TrainingDataset: generated/api/training_dataset_api.md
+      - Storage Connector: generated/api/storage_connector_api.md
+      - Feature: generated/api/feature_api.md
+      - Rule Definiton: generated/api/rule_definition_api.md
+      - Rule: generated/api/rule_api.md
+      - Expectation: generated/api/expectation_api.md
+      - Validation: generated/api/validation_api.md
+    - Contributing: CONTRIBUTING.md
+  - Hopsworks.ai: 
+    - Instroduction: hopsworksai/index.md
```
Comment (maismail) [inline mkdocs.yml:55]:
```suggestion
    - Introduction: hopsworksai/index.md
```

File: docs/hopsworksai/index.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Hopsworks.ai
+
+Hopsworks.ai is our managed platform for running Hopsworks and the Feature Store
+in the cloud. It integrates seamlessly with third-party platforms such as Databricks,
+SageMaker and KubeFlow.
```
Comment (maismail) [inline docs/hopsworksai/index.md:5]:
```suggestion
SageMaker, and KubeFlow.
```

File: docs/hopsworksai/index.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+To get started with deploying a cluster in you Azure or AWS environment, follow these guides: [Azure](azure/getting_started.md), [AWS](aws/getting_started.md).
+
+If you need more details about hopsworks.ai cluster creation steps you can check the following guides: [Azure](azure/cluster_creation.md), [AWS](aws/cluster_creation.md).
+
+## Limiting permissions
+
+Hopsworks.ai requires a set of permissions to be able to manage resources in the user’s cloud account. By default, these permissions are set to easily allow a wide range of different configurations and allow us to automate as many steps as possible. While we ensure to never access resources we shouldn’t, we do understand that this might not be enough for your organization or security policy. The following guides explain how you can reduce the permissions you give to hopsworks.ai by doing some of the cluster creation steps manually:
+
+* [Azure](azure/restrictive_permissions.md)
```
Comment (maismail) [inline docs/hopsworksai/index.md:17]: I guess we should be consistent on listing of Azure and AWS 
https://github.com/logicalclocks/feature-store-api/pull/257/files#diff-bef27b5191049cfac9fd10b40901df0b482a80decc5481ae15696b9ce93f5c33R9
https://github.com/logicalclocks/feature-store-api/pull/257/files#diff-bef27b5191049cfac9fd10b40901df0b482a80decc5481ae15696b9ce93f5c33R11

**Review by maismail — APPROVED** [review]

File: mkdocs.yml [hunk 4]
```diff
@@ -9,9 +9,50 @@ repo_url: https://github.com/logicalclocks/feature-store-api
 edit_uri: ""
 
 nav:
-  - Introduction: index.md
-  - Quickstart: quickstart.md
-  - Hopsworks.ai:
+  - Feature store:
```
Comment (moritzmeister) [inline mkdocs.yml:12]: can you capitalize the s, please :)
```suggestion
  - Feature Store: 
```

**Review by moritzmeister — CHANGES_REQUESTED** [review]
looks good, I went through all pages to see that pictures are all there.

I have to ask you for one minor change only :)

**Review by moritzmeister — APPROVED** [review]

**Pull request merged**